Tests/XCUITests/HistoryTests.swift

```diff
@@ -103,215 +98,214 @@ class HistoryTests: BaseTestCase {
         XCTAssertEqual(app.alerts.buttons["OK"].exists, false)
     }
 
-    func testRecentlyClosedOptionAvailable() {
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
-        navigator.goto(HistoryRecentlyClosed)
-        waitForNoExistence(app.tables["Recently Closed Tabs List"])
+    func testRecentlyClosedWebsiteOpen() {
+        openBookOfMozilla()
 
-        // Go to the default web site  and check whether the option is enabled
-        navigator.nowAt(LibraryPanel_History)
-        navigator.goto(HomePanelsScreen)
-        navigator.openURL(path(forTestPage: "test-mozilla-book.html"))
-        navigator.nowAt(BrowserTab)
-        navigator.goto(BrowserTabMenu)
-        // Workaround to bug 1508368
-        navigator.goto(LibraryPanel_Bookmarks)
-        navigator.goto(HomePanelsScreen)
-        navigator.performAction(Action.OpenNewTabFromTabTray)
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
+        // The tab, which is still opened, is not included in the "Recently Closed Tabs List"
+        navigator.nowAt(HomePanelsScreen)
```

Comment:
After calling openBookOfMozilla() you end up on this screen:
![Simulator Screen Shot - iPhone 14 - 2022-10-20 at 11 59 03](https://user-images.githubusercontent.com/1897507/196918825-a8d670e5-42a8-4bd8-8545-467cd88e95b4.png)

so this call is wrong, the app is not in `HomePanelScreen`, you can remove this and the next `goto(LibraryPanel_History)` will work, since the button to go to BrowserTabMenu to get to that menu is available.

Reply:
It forces the test to know where it is to continue with the next action available for that screen, but it does not take you to that screen